src/lib.rs

```diff
@@ -21,7 +21,49 @@ extern crate unicase;
 extern crate jwt;
 extern crate rand;
 
-pub mod users_db;
-pub mod users_router;
-pub mod auth_middleware;
+mod users_db;
+mod users_router;
+mod auth_middleware;
 mod errors;
+
+pub use users_db::UsersDb as UsersDb;
+pub use users_db::UserBuilder as UserBuilder;
+pub use users_db::UserBuilderError as UserBuilderError;
+pub use users_db::ReadFilter as ReadFilter;
+pub use users_db::User as User;
+pub use users_router::UsersRouter as UsersRouter;
+pub use auth_middleware::AuthMiddleware as AuthMiddleware;
+pub use auth_middleware::AuthEndpoint as AuthEndpoint;
+
+pub struct Manager {
```

Comment:
The idea is that since it is in the  module foxbox_users that calling it ````UsersManager```` instead of ````Manager```` had some sort of redundancy. But if you prefer it that way, it is fine.